src/plugins/bblogger/bblogfile.cpp

```diff
@@ -376,7 +376,7 @@ BBLogFile::repair()
     if (ftruncate(fileno(f_), fs.st_size - extra_bytes) == -1) {
       throw Exception(errno, "Failed to truncate file %s", filename_);
     }
-    all_entries_size -= extra_bytes;
+    //all_entries_size -= extra_bytes;
```

Comment:
This could also just be removed completely.

Reply:
Because if there were ever modifications after this point which do use all_entries_size this would need to be added back.